### ISSUE TITLE
Permission denied: '/path'

### ISSUE THREAD

**Issue by jjocean1234** [xgcm/xmitgcm#169]
Hi - I am a complete beginner to this software so I apologize if this is a simple or somewhat unclear question. 

I am trying to get an average U/V/Eta spatial fields (lat vs lon) for the Antarctic from the LLC4320Model simulation. In addition I would like depth vs. longitudinal cross sections of U,V, salt and potential temperature at a given latitude.  

I have got this far, but am getting a permission denied; 
![Snip20190902_5](https://user-images.githubusercontent.com/52986313/64130688-fe2fac80-cd91-11e9-81f0-6b0407187305.png)

Any help including any code to access the data and plotting would be greatly appreciated!
I have been struggling for awhile now and this data would be a good addition to an upcoming publication.

**Comment by rabernat** [comment]
Hi @jjocean1234, welcome to xmitgcm. Thanks for trying out this experimental software.

Your code is not working because the code that you are using:
```python
store = llcreader.BaseStore(fs, base_path='/path/to/global_oce_llc90')
```
is telling llcreader to look for the llc files on your local computer, at a path called `/path/to/global_oce_llc90`. Presumably you copied and pasted this code from the [documentation on manual dataset creation](https://xmitgcm.readthedocs.io/en/latest/llcreader.html#manual-dataset-creation).

It sounds like what you want is to use one of the [pre-defined datasets](https://xmitgcm.readthedocs.io/en/latest/llcreader.html#ecco-http-data-portal) from the ecco data portal. For example, to create a dataset for the surface fields U/V/Eta, you would do

```python
model = llcreader.ECCOPortalLLC4320Model()
ds = model.get_dataset(varnames=['U', 'V', 'Eta'], k_levels=[0])
```

> I am trying to get average U/V/Eta spatial fields (lat vs lon) for the Antarctic from the LLC4320Model simulation.

Can you be more specific about what you mean by "average"?

> In addition I would like depth vs. longitudinal cross sections of U,V, salt and potential temperature at a given latitude.

Because of the way the model data is stored, if you want to get a cross section, you have to basically download the full 3D fields first. And working with full-depth (i.e. 3D) output from LLC4320 can be extremely slow, due to the massive data size. Are you just looking for a single snapshot?

**Comment by jjocean1234** [comment]
Hi - thanks for the reply.
I had initially started with the code;
model = llcreader.ECCOPortalLLC4320Model()
but I receive an error message; request() got an unexpected keyword argument 'size_policy'. 

![Snip20190902_7](https://user-images.githubusercontent.com/52986313/64136131-75c70100-cdbd-11e9-854b-0f72e5220b15.png)

Yes -  for the cross sections, I am looking for a snapshot cross section.

**Comment by jjocean1234** [comment]
Hi - thanks for the reply.
I had initially started with the code;
model = llcreader.ECCOPortalLLC4320Model()
but I receive an error message; request() got an unexpected keyword argument 'size_policy'. 

![Snip20190902_6](https://user-images.githubusercontent.com/52986313/64136299-a78c9780-cdbe-11e9-93a6-4a82f3982cb2.png)
![Snip20190902_7](https://user-images.githubusercontent.com/52986313/64136305-afe4d280-cdbe-11e9-84bf-9d6947ffa314.png)

**Can you be more specific about what you mean by "average"?**
I mean average over time; presumably over a year if possible. 

**Are you just looking for a single snapshot?**
Yes -  for the cross sections, I am looking for a snapshot cross section within a specific area of the Antarctic shelf similar to this figure from Stewart et al., 2018. 
![Snip20190902_8](https://user-images.githubusercontent.com/52986313/64136246-66948300-cdbe-11e9-8c08-e6fa990d428e.png)

**Comment by rabernat** [comment]
> request() got an unexpected keyword argument 'size_policy'.

Yes, this is an unfortunate bug that just appeared due to a regression in fsspec (https://github.com/intake/filesystem_spec/issues/123). We will have a bug fix out for this very soon. In the meantime, you can fix it by downgrading fsspec:
```python
pip install fsspec==0.4.1
```

Sorry for the friction. I know this must seem annoying.

> I mean average over time; presumably over a year if possible.

So this requires you to read hundreds of TB of data from the ecco data portal. Not really feasible.

> Yes - for the cross sections, I am looking for a snapshot cross section

This should be do-able, but you'll also need the grid information, which is not yet exposed via llcreader (see #166).

We clearly have some work to do here. We appreciate your feedback and would welcome your contributions if you want to work on any of these issues.

**Comment by rabernat** [comment]
I think this is the same as #175. @jjocean1234 - have you been able to resolve this issue?
